index.html

```diff
@@ -0,0 +1,213 @@
+                              <p>Javascript</p>
+                        </div>
+                        <button>See Project</button>
+                  </div>
+            </section>
+            <section class="footer-section">
+                  <h1 class="name">About <br> Myself</h1>
+                  <p class="description">I’m a software developer! I can help you build a product , feature or
+                        website
+                        Look through some of my work and experience! If you like what you see and have a
+                        project you need coded, don’t hestiate to contact me.</p>
+                  <h4 class="connect">LET’S CONNECT</h4>
+                  <ul class="icons">
+                        <li>
+                              <a href="https://twitter.com/BlossomAniedi"><img src="./image/twitter.png"
+                                          alt="twitter icon"></a>
+                        </li>
+                        <li>
+                              <a href="https://www.linkedin.com/in/blossom-aniedi-482b84240/"><img
+                                          src="./image/linkedin.png" alt="linkedin icon" /></a>
+                        </li>
+                        <li>
+                              <a href="#"><img src="./image/meduim.png" alt="medium icon" /></a>
+                        </li>
+                        <li>
+                              <a href="https://github.com/Blossom-Aniedi"><img src="./image/git.png"
+                                          alt="git icon" /></a>
+                        </li>
+                        <li>
+                              <a href="#"><img src="./image/angelist.png" alt="angelist icon" /></a>
+                        </li>
+                  </ul>
+                  <button class="button">Get my resume</button>
+                  <section class="languages">
+                        <div class="paragraph">
+                              <p>Languages</p>
+                              <img src="./image/chevrondown.png" alt="">
+                        </div>
+                        <div class="language">
+                              <div class="javascript">
+                                    <img src="./image/javascript.png" alt="">
+                                    <p>JavaSript</p>
+                              </div>
+                              <div class="javascript">
+                                    <img src="./image/html.png" alt="">
+                                    <p>HTML</p>
+                              </div>
+                              <div class="javascript">
+                                    <img src="./image/css.png" alt="">
+                                    <p>CSS</p>
+                              </div>
+                              <div class="framework">
+                                    <h4>Frameworks</h4>
+                                    <img src="./image/chevronright.png" alt="">
```

Comment:
- [ ] I think you should consider wrapping the list of technology categories ("Languages", "Frameworks" and "Skills")  in a `ul`, each of them in a `li` which is good for browser accessibility and also for screen readers.